.travis.yml

```diff
@@ -18,7 +18,7 @@ before_install:
     - conda update --all --yes
     - travis_retry conda create --yes -n TEST python=$TRAVIS_PYTHON_VERSION --file requirements.txt
     - source activate TEST
-    - travis_retry conda install --yes --file requirements-dev.txt
+    - travis_retry pip install -r requirements-dev.txt
```

Comment:
Right now we rely on conda for the heavy weight dependencies listed in `requirements.txt`.  As soon as I get some time I will package the light weight (pure python and to easy pip install) from `requirements-dev.txt` on `conda-forge` and we can use a conda only solution.  In the end it does not matter much for Travis-CI as coda handle `pip installs` of pure package without a problem.